pkg/instrumentation/podmutator_test.go

```diff
@@ -51,31 +51,31 @@ func TestGetInstrumentationInstanceFromNameSpaceDefault(t *testing.T) {
 			Java: v1alpha1.Java{
 				Image: defaultJavaInstrumentationImage,
 				Env: []corev1.EnvVar{
-					{Name: otelSampleEnabledKey, Value: otelSampleEnabledDefaultValue}, //TODO: remove in favor of new name once safe
-					{Name: otelAppSignalsEnabledKey, Value: otelAppSignalsEnabledDefaultValue},
-					{Name: otelTracesSamplerArgKey, Value: otelTracesSamplerArgDefaultValue},
-					{Name: otelTracesSamplerKey, Value: otelTracesSamplerDefaultValue},
-					{Name: otelExporterOtlpProtocolKey, Value: otelExporterOtlpProtocolValue},
-					{Name: otelExporterTracesEndpointKey, Value: otelExporterTracesEndpointDefaultValue},
-					{Name: otelExporterSmpEndpointKey, Value: otelExporterSmpEndpointDefaultValue}, //TODO: remove in favor of new name once safe
-					{Name: otelExporterAppSignalsEndpointKey, Value: otelExporterAppSignalsEndpointDefaultValue},
-					{Name: otelExporterMetricKey, Value: otelExporterMetricDefaultValue},
-					{Name: otelExporterLogsKey, Value: otelExporterLogsDefaultValue},
+					{Name: "OTEL_SMP_ENABLED", Value: "true"}, //TODO: remove in favor of new name once safe
```

Comment:
Are all these block of EnvVar constant? If so we should consolidate them to a default const